portal-frontend/src/app/app.module.ts

```diff
@@ -48,6 +58,14 @@ import { ViewApplicationComponent } from './features/view-application/view-appli
     FileSizePipe,
     ConfirmationDialogComponent,
     ViewApplicationComponent,
+    ReviewApplicationComponent,
```

Comment:
Should I move ApplicationReview into its own module? 🤔

Reply:
TBF, it is its' own module in the backend. I support the effort.

Reply:
Will do in next MR :)